docs/UserGuide.md

```diff
@@ -45,7 +89,7 @@ management tasks done faster than traditional GUI apps.
 * Items with `…`​ after them can be used multiple times including zero times.<br>
```

Comment:
Is there any part mentioning that '[]' equals to optional? because even though tag has '...', in edit for example most of our parameters only have '[]' :)

Reply:
Yes. This has been included in the existing user guide and I did not make any changes to it. Therefore, it is not highlighted in this pull request as a change.